### ISSUE TITLE
Make collections info panel compact

### ISSUE THREAD

**Issue by 9inpachi** [HSF/phoenix#206]
The idea is to make the Collections table have a different layout for mobile / smaller views.

This is the current view of the collections info panel.

![image](https://user-images.githubusercontent.com/36920441/105216671-b1aa6400-5b74-11eb-87d5-187ce1cdf04c.png)

On smaller screens (and small panel size), the data gets cut off and we have to scroll horizontally.
It would be great if we could have a different layout on smaller view. (Maybe something where each column is on a separate line)

**Comment by ksraj123** [comment]
Seems like a good issue to start with. Looking into this. Thanks

**Comment by karthikshetty03** [comment]
@9inpachi Can I work on this issue ?

**Comment by 9inpachi** [comment]
Yes sure. Just give it a try and create a pull request if it works.

**Comment by karthikshetty03** [comment]
@9inpachi  
Can you please give me the link to the location of the code or give me some more information regarding this issue?
Also, can you tell me where can I find this page in the [online version](https://hepsoftwarefoundation.org/phoenix/)?

**Comment by 9inpachi** [comment]
You can find it if you go to an experiment from the main page (for example,
https://hepsoftwarefoundation.org/phoenix/#/atlas) and click on the "i"
icon in the bottom UI menu to open the collections info panel.

Also please take a look at `guides/users.md` to understand how everything
works.

If this information is not enough to know the code location then let me
know. :)

**Comment by karthikshetty03** [comment]
Yes, I got the code location and will try implementing the functionality, Thank You!

**Comment by karthikshetty03** [comment]
According to your suggestion, the UI on small screens should be such that the columns are one below the other, right?

**Comment by 9inpachi** [comment]
Indeed.

**Comment by karthikshetty03** [comment]
![Screenshot from 2021-02-26 15-59-18](https://user-images.githubusercontent.com/45932358/109289024-9dc8e080-784b-11eb-9b7d-2db23b318b0d.png)
@9inpachi  Is this fine?

**Comment by 9inpachi** [comment]
@karthikshetty03 Good progress so far! Thanks.

The next step would be to show the column name for each value at the left. Something like.

**columnName1:** The value
**columnName2:** Some other value

Also it'd be interesting if it worked with resizing the overlay on desktops as well.

**Comment by karthikshetty03** [comment]
> @karthikshetty03 Good progress so far! Thanks.
> 
> The next step would be to show the column name for each value at the left. Something like.
> 
> **columnName1:** The value
> **columnName2:** Some other value
> 
Yeah, will do that

**Comment by karthikshetty03** [comment]
> Also it'd be interesting if it worked with resizing the overlay on desktops as well.

@9inpachi As in if the table requires a horizontal scroll on the desktop, then I have to display all columns one below the other as well?

**Comment by 9inpachi** [comment]
Yes exactly.

**Comment by karthikshetty03** [comment]
@9inpachi Apologies for the delay, I was busy with my college exams for a week.
I resumed working on this but am stuck on the part on how to determine whether there is a horizontal scrollbar or not.
Like if there is a scrollbar, then the layout will be changed to a small screen view (each column on a separate line)
I tried looking around the overlay-component.ts file, where the overlay window resizing is handled but the clientWidth and scrollWidth of the overlayCard do not seem to be of much help.
Any hints or suggestions on how to go about this?

**Comment by 9inpachi** [comment]
Not a problem.

Could you open up a pull request so I can take a look (probably tomorrow)
at your approach so far? I can comment or direct better after that.

**Comment by karthikshetty03** [comment]
I used media queries that changes the layout for small screens.

> Could you open up a pull request so I can take a look (probably tomorrow) at your approach so far?

Yeah, will do that.

**Comment by karthikshetty03** [comment]
![Screenshot from 2021-03-10 12-35-11](https://user-images.githubusercontent.com/45932358/110591552-35f58c80-819f-11eb-9295-0cee537bf9d4.png)

@9inpachi  this works fine for mobile/smaller views.
Need to work on resizing the overlay on desktops.
I have opened a PR: #228

**Comment by AyushABD-17** [comment]
> Yes, I got the code location and will try implementing the functionality. Thank You!

can you share the code location please

**Comment by staru09** [comment]
hello can I work on this?

**Comment by EdwardMoyse** [comment]
> hello can I work on this?
> 
> 

Absolutely! Feel free to ask us with any questions.

**Comment by ishqDehlvi** [comment]
is it open to work???

**Comment by EdwardMoyse** [comment]
Not sure. @staru09 are you taking this? Please look at the earlier PR too.

**Comment by StoneCoder123** [comment]
Is this open to work on?

**Comment by EdwardMoyse** [comment]
This is still open.

**Comment by EdwardMoyse** [comment]
Yes, please do!

**Comment by Pratz2005** [comment]
![Screenshot 2024-12-12 233714](https://github.com/user-attachments/assets/6ab18759-f9ff-474c-9688-f7e21220f9eb)


does this look fine?

**Comment by EdwardMoyse** [comment]
It does - thanks!

**Comment by Pratz2005** [comment]
ok ill make the PR

**Comment by Pratz2005** [comment]
Hello, whats the status of the PR?

**Comment by FARDEEN-785** [comment]
hey How to get started or contribute is there anything i can do

**Comment by EdwardMoyse** [comment]
> hey How to get started or contribute is there anything i can do 

Hi, sorry this one should have been closed.
